code/modules/mob/living/simple_animal/hostile/true_changeling.dm

```diff
@@ -0,0 +1,141 @@
+#define TRUE_CHANGELING_REFORM_THRESHOLD 180 //3 minutes by default
+#define TRUE_CHANGELING_PASSIVE_HEAL 3 //Amount of brute damage restored per tick
+
+//Changelings in their true form.
+//Massive health and damage, but move slowly.
+
+/mob/living/simple_animal/hostile/true_changeling
+	name = "true changeling"
+	real_name = "true changeling"
+	desc = "Holy shit, what the fuck is that thing?!"
+	speak_emote = list("says with one of its faces")
+	emote_hear = list("says with one of its faces")
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "horror"
+	icon_living = "horror"
+	icon_dead = "horror_dead"
+	speed = 2
+	a_intent = "harm"
+	stop_automated_movement = 1
+	status_flags = CANPUSH
+	ventcrawler = 2
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxHealth = 500 //Very durable
+	health = 500
+	healable = 0
+	environment_smash = 1
+	melee_damage_lower = 30
+	melee_damage_upper = 30
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_MINIMUM
+	wander = 0
+	attacktext = "rips into"
+	attack_sound = 'sound/effects/blobattack.ogg'
+	next_move_modifier = 0.5 //Faster attacks
+	butcher_results = list(/obj/item/weapon/reagent_containers/food/snacks/meat/slab/human = 15) //It's a pretty big dude. Actually killing one is a feat.
+	gold_core_spawnable = 1 //To be fair, they won't be able to become changelings, like headslugs can
+	var/time_spent_as_true = 0
+	var/playstyle_string = "<b><font size=3 color='red'>We have entered our true form!</font> We are unbelievably powerful, and regenerate life at a steady rate. However, most of \
+	our abilities are useless in this form, and we must utilise the abilities that we have gained as a result of our transformation. Currently, we are incapable of returning to a human. \
+	After several minutes, we will once again be able to revert into a human. Taking too much damage will also turn us back into a human in addition to knocking us out for a long time.</b>"
+	var/mob/living/carbon/human/stored_changeling = null //The changeling that transformed
+	var/devouring = FALSE //If the true changeling is currently devouring a human
+
+/mob/living/simple_animal/hostile/true_changeling/New()
+	..()
+	spawn(0)
+		src << playstyle_string
+
+/mob/living/simple_animal/hostile/true_changeling/Life()
+	..()
+	adjustBruteLoss(-TRUE_CHANGELING_PASSIVE_HEAL) //True changelings slowly regenerate
+	time_spent_as_true += 1 //Used for re-forming
```

Comment:
Also, life ticks don't match up well with actual time so this three minute limit is more like 4 or so.